styletesla.css

```diff
@@ -1,97 +1,640 @@
 .body {
-    max-width: 1440px;
+	max-width: 1440px;
     margin: 0 auto;
     }
 .mainpage {
 	height: 100vh;
-        }
+    }
 .header {
-	height: 100px;
+	height: 10vh;
 	display: flex;
 	justify-content: space-between;
 	align-items: center;
-}
+	/*margin-bottom: 40px;*/
+    }
+ .logo {
+    padding-left: 47px;
+    }
 .navigation a {
-	display: inline;
-    margin-bottom: 67px;
+	font: 15px/18px montserratsemibold;
     cursor: pointer;
     margin-right: 44px;
-}
+    }
 .navigation__link_active {
-	border-bottom: 1px solid red;
-	margin-right: 63px;
-	color: red;}
-
+	padding-bottom: 5px;
+	border-bottom: 1px solid #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)